Tests/TuistCoreTests/Simulator/SimulatorControllerTests.swift

```diff
@@ -256,84 +283,126 @@ final class SimulatorControllerTests: TuistUnitTestCase {
         // Then
         XCTAssertTrue(system.called(launchAppCommand))
     }
-
-    private func createSystemStubs(devices: Bool, runtimes: Bool) -> SimulatorDeviceAndRuntime {
-        let bundlePath = "/path/to/bundle"
-        let buildVersion = "buildVersion"
-        let runtimeRoot = "/path/to/runtime/root"
-        let identifier = "com.apple.CoreSimulator.SimRuntime.iOS-14-4"
-        let version = "14.4"
-        let runtimeName = "iOS 14.4"
-        let runtimesJsonResponse = """
-        {
-          "runtimes" : [
-            {
-              "bundlePath" : "\(bundlePath)",
-              "buildversion" : "\(buildVersion)",
-              "runtimeRoot" : "\(runtimeRoot)",
-              "identifier" : "\(identifier)",
-              "version" : "\(version)",
-              "isAvailable" : true,
-              "supportedDeviceTypes" : [],
-              "name" : "\(runtimeName)"
-            }
-          ]
-        }
-        """
-        let dataPath = "/path/to/sim/81F0475F-0A03-4742-92D7-D59ACE3A5895/data"
-        let logPath = "/path/to/logs/81F0475F-0A03-4742-92D7-D59ACE3A5895"
-        let udid = "81F0475F-0A03-4742-92D7-D59ACE3A5895"
-        let deviceTypeIdentifier = "com.apple.CoreSimulator.SimDeviceType.iPhone-11"
-        let state = "Shutdown"
-        let deviceName = "iPhone 11"
-        let runTimeIdentifier = "com.apple.CoreSimulator.SimRuntime.iOS-14-4"
-        let devicesJsonResponse = """
-        {
-          "devices" : {
-            "\(runTimeIdentifier)" : [
-              {
-                "dataPath" : "\(dataPath)",
-                "logPath" : "\(logPath)",
-                "udid" : "\(udid)",
-                "isAvailable" : true,
-                "deviceTypeIdentifier" : "\(deviceTypeIdentifier)",
-                "state" : "\(state)",
-                "name" : "\(deviceName)"
-              }
-            ]
-          }
-        }
-        """
-
+    
+    private func createSystemStubs(devices: Bool, runtimes: Bool, versions: [SimulatorRuntimeVersion] = [.init(major: 14, minor: 4)]) -> [SimulatorDeviceAndRuntime] {
```

Comment:
`SimulatorDeviceAndRuntime` doesn't conform to `Codable`, method was manually changed to support adding more runtimes and simulators